src/components/Link/index.js

```diff
@@ -1,20 +1,31 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { Link as GatsbyLink } from 'gatsby'
 import { ExternalLink } from 'components/Icons/Icons'
+import { useValues } from 'kea'
+import { posthogAnalyticsLogic } from 'logic/posthogAnalyticsLogic'
+import { appendGclid } from 'lib/utils'
 
 export default function Link({
     to,
     children,
     className = '',
-    onClick,
-    disablePrefetch,
-    external,
+    onClick = undefined,
```

Comment:
Defaults added so that TypeScript doesn't complain about missing props on instances